collections/dominode-resources/checklists/legacy-raster.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "DomiNode - legacy raster validation checklist",
+  "description": "A checklist for validating legacy raster datasets",
+  "dataset_type": "raster",
+  "validation_artifact_type": "dataset",
+  "checks": [
+    {
+      "name": "legacy",
+      "description": "Check for legacy raster datasets.",
+      "guide": "This check will always defaults to True, it hasq been added for the validation of validation of those datasets that were previously on the legacy DomiNode",
+      "automation": {
+        "algorithm_id": "script:legacy",
+        "artifact_parameter_name": "INPUT",
+        "output_name": "Result"
```

Comment:
This should be the name you used to define your algorithm's output (check line 14 of the `processing/legacy.py` file below)